hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -63,6 +63,9 @@
       + " exists to support backward compatibility. If you use hoodie 0.3.x, do not set this parameter")
   public Boolean assumeDatePartitioning = false;
 
+  @Parameter(names = "--use-aws-glue-meta", description = "If enabled, the hudi table will get synced with the AWS Glue Meta.")
+  public Boolean isAwsGlueMetaSyncEnabled = false;
```

Comment:
minor naming thing:
i think the precise name is AWS Glue Data Catalog. https://docs.aws.amazon.com/glue/latest/dg/populate-data-catalog.html
So suggest to align argument and variable names: `--use-aws-glue-data-catalog` and `useAwsGlueDataCatalog = false`

Reply:
Please also update the copy() and toString() methods below